doc/source/user_guide/installation.rst

```diff
@@ -47,4 +50,5 @@ entered into an anaconda prompt (from scratch):
 .. code-block:: bash
 
     (base) conda activate lumispy
+    (lumispy) pip install jupyterlab
```

Comment:
It can cause issues to mix conda and pip as pip doesn't ensure ABI and it can break dependencies easily. Why recommending to use pip and not conda?